packages/jest-editor-support/src/__tests__/fixtures/snapshots/__snapshots__/react-native.example.snap

```diff
@@ -0,0 +1,105 @@
+exports[`test renders correctly 1`] = `
```

Comment:
I took two already existing tests and related snapshots from other tests in Jest, yes we can make them smaller, but I rather use a real example